index.js

```diff
@@ -52,3 +52,31 @@ module.exports.set = function (obj, path, value) {
 		obj = obj[p];
 	}
 };
+
+module.exports.del = function (obj, path) {
```

Comment:
I agree with you, but just a minor sidenote. It will probably not work on older browsers (with browserify/webpack) as `delete` is a reserved keyword. Don't think you are interested in supporting that but just wanted to clarify that :).